lms/envs/common.py

```diff
@@ -1499,6 +1499,15 @@
 # routing key that points to it.  At the moment, the name is the same.
 BULK_EMAIL_ROUTING_KEY = HIGH_PRIORITY_QUEUE
 
+# We also define a queue for smaller jobs so that large courses don't block
+# smaller emails (see BULK_EMAIL_MIN_FOR_HIGH_QUEUE setting)
+BULK_EMAIL_ROUTING_KEY_SMALL = DEFAULT_PRIORITY_QUEUE
+
+# For emails with fewer than these number of recipients, send them through
+# a different queue to avoid large courses blocking emails that are meant to be
+# sent to self and staff
+BULK_EMAIL_MIN_FOR_HIGH_QUEUE = 100
```

Comment:
Nit:  Including "HIGH_QUEUE" in the name of the threshold is assuming the value of BULK_EMAIL_ROUTING_KEY, which could be changed in future.  And "SMALL" isn't as descriptive as to what is small (i.e. is it the length of the email or the number).  I might suggest slightly different names:   BULK_EMAIL_NONBULK_ROUTING_KEY and BULK_EMAIL_MIN_FOR_BULK (or something like that -- maybe BULK_EMAIL_NONBULK_THRESHOLD or BULK_EMAIL_NONBULK_MAXIMUM).